detanalysis/semiautocuts.py

```diff
@@ -2203,5 +2202,11 @@ def get_randoms_passfrac_table(self):
         print(tabulate(tab_table, headers = headers_, tablefmt = 'orgtbl', floatfmt='.3f'))
 
 
+    def get_final_values(self,feature_exp):
```

Comment:
this is the same as calling df.cut_blah.values... is it worth having as a function here?

Reply:
Ok honestly wasn't sure if that would work. If you think there is already functionality for this then we don't need the new function for it